Pipfile

```diff
@@ -13,6 +13,7 @@ pytest = "*"
 pytest-cov = "*"
 pytest-freezegun = "*"
 black = "*"
+monkeytype = "*"
```

Comment:
Is this the sort of thing we want to be running manually regularly? If so, maybe add something to the readme about it?

Reply:
Yeah this is a tool I'm trying which creates type annotations automatically - I'll add something in the README.